### PR TITLE
Sass migration + refactoring

### PR DESCRIPTION
**Pull request by sjv01180**
Refactored index.scss for improved readability and DRY principles.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+node_modules
```
Comment (ww2497) [inline .gitignore:24]: It does this already, see line 4

File: src/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.scss]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```
Comment (peterkos) [inline src/.editorconfig:8]: Should be newline at end of file here, and also, `insert_final+newline = true`

File: src/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,8 @@
+root = true
```
Comment (peterkos) [inline src/.editorconfig:1]: Is `root` project root or `src` root?

File: src/index.scss [hunk 4]
```diff
@@ -55,36 +99,76 @@ html, body, #root, #app, .full-height{
...
-.navigation ul li a{
-  color: $blue;
+  ul{
+    list-style-type: none;
+    float: right;
+  
+    margin: 0;
+    padding: 0;
+
+    li{
```
Comment (peterkos) [inline src/index.scss:110]: All CSS I've ever seen _always_ puts a space after the selector -- just a small style thing

File: .gitignore [hunk 5]
```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```
Comment (ww2497) [inline .gitignore:24]: Just saying, I think we've decided to use npm instead of yarn to avoid dependency conflicts

File: .editorconfig [hunk 6]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[./src/*.scss]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = true
```
Comment (peterkos) [inline .editorconfig:8]: This file still needs a newline at the end

File: .gitignore (hunk 5 above)
Comment (peterkos) [reply .gitignore:24]: Ok!

File: src/index.scss [hunk 7]
```diff
@@ -55,68 +99,134 @@ html, body, #root, #app, .full-height{
...
+  }
 }
-.left-half, .right-half{
+.right-half {
   flex: 50%;
 }
-.left-half{
-  margin-bottom: 2.5vh;
+.left-half {
+    @extend .right-half;
```
Comment (ww2497) [inline src/index.scss:177]: Originally, left-half was like flex: 40% and right-half had flex: 60% so the hero-img was larger but I decided that it was unnecessary. They're meant to be imbalanced if they're not both at 50% though

File: src/index.scss [hunk 8]
```diff
@@ -160,144 +284,46 @@ html, body, #root, #app, .full-height{
...
-tablet & mobile have the column layout
-tablet has fixed widths and font sizes
-mobile has scaling widths and font sizes
-*/
-
-/*non-desktop*/
-@media (max-width: 1100px){
-  .mobile-hide{
+.mobile-banner {
     display: none;
```
Comment (ww2497) [inline src/index.scss:313]: I don't believe this is necessary, it should already have a desktop-hide class

File: src/index.scss [hunk 9]
```diff
@@ -1,30 +1,61 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900;1000&display=swap');
 
 $blue: #465491;
+$white: #fff;
```
Comment (ww2497) [inline src/index.scss:4]: We've decided to use just 'white' which browsers support instead of a variable for white

File: src/index.scss [hunk 10]
```diff
@@ -160,144 +284,46 @@ html, body, #root, #app, .full-height{
...
+
+        font-size: 1rem;
+        font-weight: 400;
+    }
+
+    @include mobile {
+        font-size: 3.5vw;
+        font-weight: 500;
+    }
 }
```
Comment (peterkos) [inline src/index.scss:329]: newline @ end?

File: src/index.scss (hunk 9 above)
Comment (peterkos) [reply src/index.scss:4]: Okay! Could this be reflected in this PR?

**Pull request merged**